pyprof/prof/activation.py

```diff
@@ -51,37 +49,28 @@ def __init__(self, d):
         arg = args[0]
         assert (arg['type'] == "tensor")
 
-        self.i = arg
+        self.inp = Tensor(arg['shape'], arg['dtype'])
```

Comment:
`inp` is not known or descriptive. Please rename for readability. `index` perhaps?